src/amp-story-player/amp-story-player-impl.js

```diff
@@ -211,6 +212,30 @@ export class AmpStoryPlayer {
     shadowRoot.appendChild(this.rootEl_);
   }
 
+  /**
+   * Helper to create a button.
+   * @private
+   */
+  initializeButton_() {
+    const option = this.element_.getAttribute('button');
+    if (!option || (option !== 'close' && option !== 'back')) {
+      return;
+    }
+    const optionBool = option === 'close';
+
+    const button = this.doc_.createElement('a');
+    button.classList.add('amp-story-player-' + option + '-button');
+    button.id = option + '-button';
```

Comment:
This might interfere with the publisher's own elements, we should avoid setting `id` to the elements.

Reply:
This was just for testing, but I think I can change it to be querySelector("a") and remove this line.